### PR TITLE
feat: add g:previm_custom_preview_base_dir

### PR DESCRIPTION
**Pull request by muxueqz**
Make a new PR because #168

### CODE REVIEW

**Comment by mattn** [comment]
Thank you. Will merge in later.

File: autoload/previm.vim [hunk 1]
```diff
@@ -96,8 +97,25 @@ endfunction
 
 let s:base_dir = fnamemodify(expand('<sfile>:p:h') . '/../preview', ':p')
 
+function! s:fix_preview_base_dir() abort
+  if exists('g:previm_custom_preview_base_dir')
+    let s:preview_base_dir = expand(g:previm_custom_preview_base_dir)
+    if !isdirectory(s:preview_base_dir . '_/js/previm.js')
```
Comment (mattn) [inline autoload/previm.vim:103]: filereadable?

Comment (mattn) [reply autoload/previm.vim:103]: or something using preview_base_dir (at least, it should NOT be preview.js)

Comment (muxueqz) [reply autoload/previm.vim:103]: > filereadable?

yes. I'll fix it

**Comment by mattn** [comment]
Thank you

**Pull request merged**